src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html

```diff
@@ -11,34 +11,8 @@ <h2 class="utrecht-heading-2 {% if userfeed.action_required %}indicator{% endif
         </div>
     </header>
 
-    <div class="userfeed__summary">
-        <ul class="userfeed__list">
-            {% for line in userfeed.summary %}
-            <li class="userfeed__list-item">
-                <p class="userfeed__list-paragraph">{{ line }}</p>
-            </li>
-            {% endfor %}
-        </ul>
-    </div>
-    <div class="card-container card-container--columns-2 plugin-card">
-        {% for item in userfeed.items %}
-            <a href="{{ item.action_url }}" class="card card--status card--status--{{ item.status_indicator }} {% if item.is_completed %}card--completed{% endif %}">
-                <div class="userfeed__marker userfeed__marker--{{ item.status_indicator }}">{# ! Space for when item.status_text is empty #}</div>
-
-                <div class="card__body card__body--tabled">
-                    <h3 class="utrecht-heading-3 userfeed__title">{{ item.title }}</h3>
-                    {% if item.status_text %}
-                        {% include "components/StatusIndicator/StatusIndicator.html" with status_indicator=item.status_indicator status_indicator_text=item.status_text %}
-                    {% endif %}
-                    <p class="userfeed-card__description">
-                        <span class="status">{{ item.message }}</span>
-                    </p>
-                    <span class="button button--icon-before button--transparent">
-                    {% icon icon="east" icon_position="after" primary=True outlined=True %}
-                    </span>
-                </div>
-            </a>
-        {% endfor %}
+    <div class="card-container card-container--columns-1 plugin-card">
+        <action-list actions='{{ userfeed_item_list_json|safe }}' />
```

Comment:
The JSON should be escaped, it'll be un-escaped in the React component:

```suggestion
        <action-list actions='{{ userfeed_item_list_json }}' />
```